plugins/woocommerce-admin/client/products/fields/variations/variations.scss

```
@@ -36,4 +44,8 @@
 		position: absolute;
 		margin: 0;
 	}
+
+	&__footer {
+		padding: $gap-small $gap;
```

One more very minor point I missed before- I think this might be:
```suggestion
		padding: $gap;
```

Which gets the height to `68px` to match the design.

---

Thanks!!!!